jsgtk_modules/fs.js

```diff
@@ -134,14 +134,21 @@ const
   }),
   fs = {
     readFile: function readFile(file, options, callback) {
-      // TODO: supports options
-      if (!callback) callback = options;
+      // TODO: support the 'flag' option
+      let encoding = options && options.encoding || options;
```

Comment:
if options is passed but there's no flag `encoding` the `encoding` will point to an object and be truthy. This make the logic further down fail.